docs/tools/aiven-console.rst

```diff
@@ -38,22 +38,22 @@ Payment cards
 """""""""""""
 In the **Payment Options** section, you can :doc:`add or update credit card details </docs/platform/howto/manage-payment-card>`, assign projects to a payment method, and update the :doc:`billing contact </docs/platform/howto/change-billing-contact>`.
 
-Account settings
------------------
+Organization and organizational unit settings
+----------------------------------------------
 
-The account you are currently working with is displayed at the top of the page. You can switch accounts by clicking the account name to open the drop-down menu. You can also create a new account from this menu.
+The organization or organizational unit that you are currently working with is displayed at the top of the page. You can switch to another organization or organizational unit by clicking the name to open the drop-down menu. You can also create a new organization from this menu.
 
-If you don't have an account, click **Create account** to :doc:`create your first account</docs/tools/aiven-console/howto/create-accounts>`. 
+If you don't have an organization, click **Create organization** to :doc:`create your first organization</docs/tools/aiven-console/howto/create-accounts>`. 
  
-.. note:: We strongly recommend creating an account. It makes managing your projects much easier and comes with many additional features, such as teams (user groups), billing groups, and SAML authentication.
+.. note:: We strongly recommend creating an organization. It makes managing your projects much easier and comes with many additional features, such as teams (user groups), billing groups, and SAML authentication.
```

Comment:
Is there a stronger version of "strongly recommend "?

Reply:
Off the top of my head, not really. That is, not without sounding like we are scolding the users. :) But, we could spell out the consequences of not doing it in addition to listing the benefits?